Assets/Scripts/Crab.cs

```diff
@@ -0,0 +1,154 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Crab : MonoBehaviour
+{
+    const string LEFT = "left";
+    const string RIGHT = "right";
+
+    [SerializeField]
+    Transform _castPos;
+    [SerializeField]
+    float _baseHorizontalCastDist;
+    [SerializeField]
+    float _baseVerticalCastDist;
+    [SerializeField]
+    float playerFollowDist;
+    [SerializeField]
+    Transform _playersTransform;
+    [SerializeField]
+    bool keepSceneViewActive;
+
+    string _facingDirection;
+    Rigidbody2D _rb2d;
+    Animator _anim;
+
+    readonly float _moveSpeed = 0.3f;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        _facingDirection = RIGHT;
+        _rb2d = GetComponent<Rigidbody2D>();
+        _anim = GetComponent<Animator>();
+
+        // kad nepermestu i game ekrana, o liktu scene
+        if (keepSceneViewActive && Application.isEditor)
+        {
+            UnityEditor.SceneView.FocusWindowIfItsOpen(typeof(UnityEditor.SceneView));
+        }
```

Comment:
It's a hacky solution, idk how it should be moved to another class. I'll just remove this and keep it locally because maybe others don't want to stay in the scene view on game start.